src/SFA.DAS.QnA.Api/web.config

```diff
@@ -1,25 +1,31 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-    <location path="." inheritInChildApplications="false">
-        <system.webServer>
-            <security>
-                <requestFiltering allowDoubleEscaping="true" />
-            </security>
-            <handlers>
-                <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
-            </handlers>
-            <aspNetCore processPath="dotnet" arguments=".\SFA.DAS.QnA.Api.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
-            <rewrite>
-                <rules>
-                    <rule name="Rewrite AlwaysOn" stopProcessing="true">
-                        <match url="^$" />
-                        <conditions>
-                            <add input="{HTTP_USER_AGENT}" pattern="^AlwaysOn$" />
-                        </conditions>
-                        <action type="Rewrite" url="/health" />
-                    </rule>
-                </rules>
-            </rewrite>
-        </system.webServer>
-    </location>
+  <location path="." inheritInChildApplications="false">
+    <system.webServer>
+      <security>
+        <requestFiltering allowDoubleEscaping="true" />
+      </security>
+      <handlers>
+        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
+      </handlers>
+      <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" hostingModel="InProcess">
+        <environmentVariables>
+          <environmentVariable name="ASPNETCORE_HTTPS_PORT" value="44368" />
+          <environmentVariable name="COMPLUS_ForceENC" value="1" />
+          <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Development" />
+        </environmentVariables>
+      </aspNetCore>
+      <rewrite>
+        <rules>
+          <rule name="Rewrite AlwaysOn" stopProcessing="true">
+            <match url="^$" />
+            <conditions>
+              <add input="{HTTP_USER_AGENT}" pattern="^AlwaysOn$" />
+            </conditions>
+            <action type="Rewrite" url="/health" />
+          </rule>
+        </rules>
+      </rewrite>
+    </system.webServer>
+  </location>
```

Comment:
It should be ok I think, but better make sure the testers know this needs a regression test for EPAO as well as usual tests, as this affect whole app

Reply:
Change reverted. 👍